pkg/client/monitoring/v1alpha1/alertmanager.go

```diff
@@ -26,8 +28,8 @@ import (
 )
 
 const (
-	TPRAlertmanagersKind = "Alertmanager"
-	TPRAlertmanagerName  = "alertmanagers"
+	CRDAlertmanagersKind = "Alertmanager"
```

Comment:
We should rethink the naming here as we're using it in CRD and TPR cases, maybe just "AlertmanagerKind". Same goes for all other functions, vars, etc we should get rid of the TPR/CRD in the naming if it is not actually specific to one or the other.

For example "CreateTPRs" and "CreateCRDs" are both valid namings, but where we use things in both contexts, they should be more generic.